.github/workflows/_arbitrator.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: nitro-arbitrator
+on:
+  workflow_call:
+    inputs:
+      run:
+        type: boolean
+        default: true
+    outputs:
+      ok:
+        value: ${{ jobs.aggregate.outputs.ok }}
+
+jobs:
+  arbitrator:
+    if: ${{ inputs.run }}
```

Comment:
it seems that this could/should be handled on the caller side, without passing an input

Reply:
Done